README.md

```diff
@@ -1,5 +1,7 @@
 # Lyef React Component
 [![Build Status](https://travis-ci.org/lyef/lyef-react-component.svg)](https://travis-ci.org/lyef/lyef-react-component/)
+![size](https://badge-size.herokuapp.com/lyef/lyef-react-component/master/dist/Component.min.js.svg)
+![gzip size](https://badge-size.herokuapp.com/lyef/lyef-react-component/master/dist/Component.min.js.svg?compression=gzip)
 
```

Comment:
@Nipher I really prefer to use more objetive message on `alt`. For an example:

1. `Normal component file size`
2. `Minified component file size`

But, it's my prefer 😸